app_test.go

```diff
@@ -240,8 +252,13 @@ func TestAppStart(t *testing.T) {
 			Invoke(Invoke(func(type1) {
 			})),
 		)
+		newErr := app.Err()
+		require.Error(t, newErr)
+
 		err := app.Start(context.Background())
 		require.Error(t, err, "expected start failure")
+		assert.Equal(t, err, newErr, "start should return the same error fx.New encountered")
```

Comment:
Run just panics/exits, right? I think that's preferable. Keeps the default `main()` the same.